### DISCUSSION TITLE
Join.

### DISCUSSION THREAD

**Discussion by zcns** [katorlys/HelpCenter#3] [Join Request]
### Profile Information
**Github username:*zcns*   
**Contact email:* *   

### What requirements you meet

- I will prove my value.

### Your thoughts

- I'm joining Katorly Lab because I am a Minecraft hobby developer
- I'm going to Write code and create plugins

**Comment by katorly** [comment]
Welcome! 加入邀请已发送.  

以后可以用中文（既然你是中国的），  
有空加一下 Gitter，方便聊天：https://app.gitter.im/#/room/#katorlys:gitter.im

**Reply by zcns** [reply]
OK
